src/zcl_abapgit_repo.clas.abap

```diff
@@ -295,7 +329,8 @@ CLASS ZCL_ABAPGIT_REPO IMPLEMENTATION.
     DATA: lt_updated_files TYPE zif_abapgit_definitions=>ty_file_signatures_tt,
           lx_error         TYPE REF TO zcx_abapgit_exception.
 
-    deserialize_checks( ).
+    check_write_protect( ).
```

Comment:
suggest also adding the two find_ methods

```suggestion
    find_remote_dot_abapgit( ).
    find_remote_dot_apack( ).

    check_write_protect( ).
```

find_remote_dot_abapgit is in check_language, but not calling find_remote_dot_apack might cause problems?